src/test/java/com/puppycrawl/tools/checkstyle/PackageObjectFactoryTest.java

```diff
@@ -59,6 +59,11 @@
  */
 public class PackageObjectFactoryTest {
 
+    private static final Set<String> INTERNAL_MODULES = Collections.unmodifiableSet(
+            new HashSet<>(Collections.singletonList(
+            "com.puppycrawl.tools.checkstyle.meta.JavadocMetadataScraper"
+    )));
```

Comment:
```
Collections.unmodifiableSet(
            new HashSet<>(Collections.singletonList(
```
All this can be changed to Collections.singletonSet()

Reply:
Since I moved `INTERNAL_MODULES` to `Definitions` class, it needs unmodifiable object.